src/Symfony/Bundle/FrameworkBundle/Resources/config/html_sanitizer.php

```diff
@@ -13,13 +13,18 @@
 
 use Symfony\Component\HtmlSanitizer\HtmlSanitizer;
 use Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig;
+use Symfony\Component\HtmlSanitizer\HtmlSanitizerInterface;
 
 return static function (ContainerConfigurator $container) {
     $container->services()
-        ->set('html_sanitizer.config', HtmlSanitizerConfig::class)
+        ->set('html_sanitizer.config.default', HtmlSanitizerConfig::class)
             ->call('allowSafeElements')
 
-        ->set('html_sanitizer', HtmlSanitizer::class)
+        ->set('html_sanitizer.sanitizer.default', HtmlSanitizer::class)
             ->args([service('html_sanitizer.config')])
+            ->tag('html_sanitizer', ['name' => 'default'])
```

Comment:
After upgrade to 6.1.0-RC1 I got:
![obraz](https://user-images.githubusercontent.com/16488888/168799605-235f5241-8592-49ba-a796-cbe0d485e0f3.png)

Looks like the args for HtmlSanitizer service may be invalid and IMO should be:

```suggestion
        ->set('html_sanitizer.sanitizer.default', HtmlSanitizer::class)
            ->args([service('html_sanitizer.config.default')])
            ->tag('html_sanitizer', ['name' => 'default'])
```

and it's works for me.